cb-talent-development/employer-style-base
# EM-644: Fractal Pattern Library Documentation

**Purpose:**

Additional documentation for Fractal pattern library added in #120 
* `README` updates
* Pattern library styling
* Move some `.scss` files into directories for use with Fractal
* Removed `$link-color-transition` which is no longer in use
  * We no longer change link color so the color transition wasn't doing anything

**JIRA:**
https://cb-content-enablement.atlassian.net/browse/EM-644

**Changes:**
* Changes to setup
  * no change to how style-base is used in Employer

* Architectural changes
  * Each component now comes with its own directory and a set of documentation-related files. `_base.scss` is updated accordingly

* Migrations
  *
  
* Library changes
  * will bump version

* Side effects
  * there should be none, but do check employer for weirdness

**Screenshots**
n/a

**QA Links:**
http://cb-talent-development.github.io/employer-style-base/
(You can verify that the changes have not affected Employer here: http://web.employer-1.development.c66.me/company/why-careerbuilder)

**How to Verify These Changes**
* Specific pages to visit
  * http://cb-talent-development.github.io/employer-style-base/

* Steps to take
  * Check it out!
  * More importantly check out the diffs of this PR, see if you understand what's going on, how documentation is added, how file structures have changed, and give feedback
  * Check Employer locally. Have buttons or animations changed? (they shouldn't)
  
* Responsive considerations
  * n/a


**Relevant PRs/Dependencies:**


**Additional Information**
